staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -173,6 +173,9 @@ type Plugins struct {
 	// Filter is a list of plugins that should be invoked when filtering out nodes that cannot run the Pod.
 	Filter *PluginSet `json:"filter,omitempty"`
 
+	// PostFilter is a list of plugins that are invoked after filtering phase, no matter filtering succeeds or fails.
```

Comment:
grammar nit: "no matter whether" or "whether filtering succeeds or not"

Reply:
sure, fixed.